src/init-ceph.in

```diff
@@ -740,6 +740,25 @@ if [ "$command" = "stop" -o "$command" = "onestop" ]; then
     what="$new_order"
 fi
 
+# Check if the monitors are up before starting any mds
+what_out=
```

Comment:
move this at the same level with what=what_out.

Reply:
done